irispy/tests/test_iris_tools.py

```diff
@@ -157,7 +158,8 @@ def test_uncalculate_exposure_time_correction_error(input_arrays, old_unit, expo
                                                                exposure_time, force=force)
 
 def test_get_iris_response_not_equal_to_one():
-    assert pytest.raises(TypeError, iris_tools.get_iris_response, time_obs, pre_launch=False, response_version=3)
+    assert pytest.raises(KeyError, iris_tools.get_iris_response, time_obs,
+                         pre_launch=False, response_version=13)
```

Comment:
Not sure this test is performing the same task now that it did before...

Reply:
No, I meant to ask you if you want it removed. It was there before version 3 was incorporated so I used a creative way to sort of "commenting it out." Would you like to keep it with some modifications?

Reply:
It was for ruling out version `3` or `4` when they were not supposed to have been implemented in `get_iris_response`, so the test would throw a `KeyError` originally, I changed it to `TypeError` in a previous PR to pass the local tests, if my memory is any reliable. So this test may not be necessary any more.